pkg/abi/linux/fuse.go

```diff
@@ -475,7 +522,7 @@ type FUSEReadIn struct {
 // The second part of the payload is the
 // binary bytes of the data to be written.
 //
-// +marshal
+// +marshal dynamic
```

Comment:
FUSEWriteIn is not a dynamic type. Please revert the changes to this.
The only change seems to be in SizeBytes() where you want it to also add `Size` to the total bytes.

However that seems incorrect. That payload of Size bytes is not part of this struct. Callers should call `SizeBytes` for just this struct and manually add FUSEWriteIn.Size while doing calculations that require the payload size.

Reply:
> However that seems incorrect. That payload of Size bytes is not part of this struct. Callers should call SizeBytes for just this struct and manually add FUSEWriteIn.Size while doing calculations that require the payload size.
```
req.payload = data[written : written+toWrite]
```
This is how it looks currently, it is adding payload outside of transmission packet and fuse treats it as a garbage as FUSEHeader.len remained unchanged. So it's enough to fix a length field of overall packet, what I did.
I see following options when reverting a FUSEWriteIn:
* Add another `data` parameter (payload2) into NewRequest so NewRequest would support dynamic payload.
* Add another dynamic struct like `FUSEDynamicWriteIn` with embedded FUSEWriteIn into it, to use unaltered NewRequest, so things mostly remains the same.